runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayUtil.java

```diff
@@ -92,7 +92,7 @@ public static void renderTextLocation(Graphics2D graphics, Point txtLoc, String
 		graphics.setColor(Color.BLACK);
 		graphics.drawString(text, x + 1, y + 1);
 
-		graphics.setColor(color);
+		graphics.setColor(new Color(color.getRed(), color.getGreen(), color.getBlue()));
```

Comment:
We'd honestly be better off making some kind of `ColorUtil` function along the lines of `Color colorOfAlpha(Color color, int alpha)`, as this can be used in many other places as well.

Reply:
I was thinking about something similar, just a `Color opaque(Color col)` that would remove alpha from a color. But specifying the actual alpha is more flexible :+1: